email_validation_check/models/res_partner.py

```diff
@@ -79,6 +79,11 @@ def check_exist_email(self):
                 continue
             if partner.id in ignore_partner_ids:
                 continue
+            user_parter = self.env['res.users'].search([
+                ('partner_id', '=', partner.id),
+            ])
+            if user_parter:
+                continue
```

Comment:
Please add a short comment in the code to explain it

ie:
`# Ignore check on partners with users because ... `


Also maybe instead of searching, you could do this:

```py
if partner.user_ids:
    continue
```

Reply:
Error only raised if the user is linked to a partner?

Reply:
I don't know, that's what I'm asking you :laughing: 

Is this error only happening when the partner is linked to a user?
Why?

I don't like this fix too much because it's disabling email validation for some partners :/
Isn't there another way to cleanly fix it?

Reply:
Yes. An error occurs when a user is linked to a partner.
email validation is working.
No, I couldn't find any other way to fix it.

Reply:
ok